VoodooRMI/Info.plist

```diff
@@ -50,6 +50,36 @@
 			<key>CFBundleIdentifier</key>
 			<string>com.1Revenger1.VoodooRMI</string>
 		</dict>
+		<key>RMII2CDevice</key>
+		<dict>
+			<key>Configuration</key>
+			<dict>
+				<key>MinYDiffThumbDetection</key>
+				<integer>200</integer>
+				<key>ForceTouchEmulation</key>
+				<true/>
+				<key>ForceTouchMinPressure</key>
+				<integer>90</integer>
+				<key>DisableWhileTypingTimeout</key>
+				<integer>100</integer>
+				<key>TrackstickMultiplier</key>
+				<integer>20</integer>
+				<key>TrackstickScrollMultiplierX</key>
+				<integer>20</integer>
+				<key>TrackstickScrollMultiplierY</key>
+				<integer>20</integer>
+				<key>TrackstickDeadzone</key>
+				<integer>1</integer>
+			</dict>
+			<key>IOProbeScore</key>
+			<integer>2910</integer>
+			<key>IOProviderClass</key>
+			<string>RMII2C</string>
+			<key>IOClass</key>
+			<string>RMIBus</string>
+			<key>CFBundleIdentifier</key>
+			<string>com.1Revenger1.VoodooRMI</string>
+		</dict>
```

Comment:
You should be able to just ass RMII2C as a provider class to RMIBus, and not need an extra entry. Should just be something like
```
<key>IOProviderClass</key>
<string>RMISMBus RMII2C</string>
```
I haven't tested this myself though

Reply:
It seems that the change won't work. Maybe dual personalities is necessary or we can use something like VoodooInput = true instead?

Reply:
yep that works too